crates/starknet-devnet-server/src/api/json_rpc/mod.rs

```diff
@@ -205,6 +206,37 @@ impl JsonRpcHandler {
         }
     }
 
+    async fn get_latest_block(&self) -> StarknetBlock {
+        let starknet = self.api.starknet.lock().await;
+        match starknet.get_block(&BlockId::Tag(BlockTag::Latest)) {
+            Ok(block) => block.clone(),
+            Err(_) => StarknetBlock::create_empty_accepted(),
```

Comment:
why `create_empty_accepted` is used, isnt it better to return an Option from `get_latest_block`. Otherwise notify_subscribers will push some default data to the socket

Reply:
I agree it's not ideal, but either returning a Result or an Option requires later handling of Err or None case in two places, which I thought was just unnecessary, especially since the latest block is always defined, so the Err/None case will never happen.

Reply:
Added comment in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/634/commits/9369969e6cc9fd5a47b00585b4790216e87b9479